webpack.config.js

```diff
@@ -11,6 +11,13 @@ module.exports = {
         filename: "[name].js",
         path: path.resolve(__dirname, "_site/assets/js")
     },
+    cache: {
+        type: "filesystem",
+        buildDependencies: {
+            config: [__filename]
+        },
+        cacheDirectory: path.resolve(__dirname, "node_modules/.cache/webpack")
```

Comment:
Does this seem to be consistent with the default value, and is there a need for explicit configuration?

Ref https://webpack.js.org/configuration/cache/#cachecachelocation

Reply:
Yes this is the default value. We can omit this if we want.

Reply:
If it’s the same value, I’m in favor of omitting.

Reply:
Updated.